etna/whatson/models.py

```diff
@@ -864,118 +752,28 @@ def save(self, *args, **kwargs):
     base_form_class = EventPageForm
 
 
-class EventFilterForm(forms.Form):
-    date = forms.DateField(
-        label="Choose a date",
-        required=False,
-        widget=forms.DateInput(
-            attrs={"type": "date", "class": "filters__date", "data-js-date": ""}
-        ),
-    )
-
-    event_type = forms.ModelChoiceField(
-        widget=forms.RadioSelect(
-            attrs={"class": "filters__radio", "data-js-event-type": ""}
-        ),
-        queryset=EventType.objects.all(),
-        required=False,
-        label="What",
-    )
-
-    is_online_event = forms.BooleanField(
-        label="Online",
-        required=False,
-        widget=forms.CheckboxInput(
-            attrs={"class": "filters__toggle-input", "data-js-online": ""}
-        ),
-    )
-
-    family_friendly = forms.BooleanField(
-        label="Family friendly",
-        required=False,
-        widget=forms.CheckboxInput(
-            attrs={"class": "filters__toggle-input", "data-js-family": ""}
-        ),
-    )
-
-
-class ExhibitionHighlight(Orderable):
-    """
-    This model is used to add highlights to exhibition pages.
-    """
-
-    page = ParentalKey(
-        "whatson.ExhibitionPage",
-        on_delete=models.CASCADE,
-        related_name="highlights",
-    )
-
-    title = models.CharField(
-        max_length=255,
-        verbose_name=_("title"),
-    )
-
-    image = models.ForeignKey(
-        get_image_model_string(),
-        null=True,
-        on_delete=models.SET_NULL,
-        related_name="+",
-    )
-
-    link = models.ForeignKey(
-        "wagtailcore.Page",
-        null=True,
-        on_delete=models.SET_NULL,
-        related_name="+",
-    )
-
-    panels = [
-        FieldPanel("title"),
-        FieldPanel("image"),
-        FieldPanel("link"),
-    ]
-
-
-class HeroColourChoices(models.TextChoices):
-    """
-    This model is used to allow for colour choice on the hero intro
-    for Exhibition pages.
-    """
-
-    LIGHT = "light", _("Light")
-    DARK = "dark", _("Dark")
-
-
-class ExhibitionPage(ArticleTagMixin, TopicalPageMixin, BasePageWithRequiredIntro):
+class ExhibitionPage(
+    ArticleTagMixin,
+    AccentColourMixin,
+    HeroImageMixin,
```

Comment:
Should the hero image be required for this page type? Do we need another mixin like `RequiredHeroImageMixin`?

Reply:
Missed the asterisk on the spec - fixing now!